src/frontends/pytorch/src/op/norm.cpp

```diff
@@ -326,6 +327,63 @@ OutputVector translate_frobenius_norm(const NodeContext& context) {
     return {result};
 }
 
+OutputVector translate_rms_norm(const NodeContext& context) {
+    // Tensor = aten::rms_norm(%input_data.1, %2, %4, %3)
+    num_inputs_check(context, 2, 4);
+    auto x = context.get_input(0);
+    auto normalized_shape = context.get_input(1);
+    Output<Node> eps;
+    if (!context.input_is_none(3)) {
+        eps = context.get_input(3);
+        if (eps.get_element_type().is_dynamic() || eps.get_element_type() != x.get_element_type())
+            eps = std::make_shared<v1::ConvertLike>(eps, x);
+    } else {
+        switch (x.get_element_type()) {
+        case element::bf16:
+            eps = v0::Constant::create(ov::element::bf16, {}, {std::numeric_limits<bfloat16>::epsilon()});
+            break;
+        case element::f16:
+            eps = v0::Constant::create(ov::element::f16, {}, {std::numeric_limits<float16>::epsilon()});
+            break;
+        case element::f64:
+            eps = v0::Constant::create(ov::element::f64, {}, {std::numeric_limits<double>::epsilon()});
+            break;
+        case element::f32:
+            eps = v0::Constant::create(ov::element::f32, {}, {std::numeric_limits<float>::epsilon()});
+            break;
+        default:
+            eps = v0::Constant::create(ov::element::f32, {}, {std::numeric_limits<float>::epsilon()});
+            eps = std::make_shared<v1::ConvertLike>(eps, x);
+        }
+    }
+    context.mark_output(eps);
+
+    // normalized shape represent D last dimensions to be normalized
+    auto num_axes = context.mark_node(std::make_shared<v3::ShapeOf>(normalized_shape, element::i32));
+    num_axes = context.mark_node(std::make_shared<v0::Squeeze>(num_axes));
```

Comment:
it create dynamic rank for output. I think - it is better to use Reshape operation with scalar target shape